src/Enzo/hydro-mhd/SetMinimumSupport.cpp

```diff
@@ -12,10 +12,15 @@
 //----------------------------------------------------------------------
  
 int EnzoBlock::SetMinimumSupport(enzo_float &MinimumSupportEnergyCoefficient,
+                                 enzo_float minimum_pressure_support_parameter,
 				 bool comoving_coordinates)
 {
   const int in = cello::index_static();
```

Comment:
Is this line required at all anymore?

Reply:
This argument is used  is used inside of `SetMinimumSupport`. Specifically, the argument is multiplied all of the other variables that are used to compute `MinimumSupportEnergyCoefficient`